doctoc.js

```diff
@@ -104,7 +104,7 @@ var entryPrefix = argv.entryprefix || '-';
 var processAll = argv.all;
 var stdOut = argv.s || argv.stdout
 var updateOnly = argv.u || argv['update-only']
-
+var syntax = argv['syntax'] || "md"
```

Comment:
Run a quick check here to validate correctness, since we don't seem to get safety automatically, something like

```js
if (!SUPPORTED_EXTENSIONS.includes(syntax) throw new Error(...)
```

This will help prevent mistakes from users.